README.rst

```diff
@@ -93,23 +88,57 @@ To allow for runtime specification of keys you can optionally pass the
         private_key='98dfg6df7g56df6gdfgdfg65JHJH656565GFGFGs',
     )
 
-If specified these parameters will be used instead of your reCAPTCHA project
-settings.
+If specified these parameters will be used instead of your reCAPTCHA project settings.
+
+Widgets
+~~~~~~~
+
+There are two widgets that can be used with the ``ReCaptchaField``;
+
+    ``ReCaptchaV2Checkbox`` for Google reCAPTCHA V2 - Checkbox
```

Comment:
It would be nice to make "Google reCAPTCHA V2 - Checkbox" a link to Google's documentation for this.